velox/dwio/dwrf/utils/test/ProtoUtilsTests.cpp

```diff
@@ -40,17 +40,6 @@ TEST(ProtoUtilsTests, AllTypes) {
   }
 }
 
-TEST(ProtoUtilsTests, UnionDeprecation) {
```

Comment:
This is moved to `HiveTypeParserTests` since one such test is already present there.